includes/admin/class-pinterest-for-woocommerce-admin.php

```diff
@@ -398,31 +398,55 @@ private function get_component_settings() {
 						admin_url( 'admin.php' )
 					)
 				),
-				'domainToVerify'  => wp_parse_url( site_url(), PHP_URL_HOST ),
-				'isConnected'     => ! empty( Pinterest_For_Woocommerce()::get_token()['access_token'] ),
-				'apiRoute'        => PINTEREST_FOR_WOOCOMMERCE_API_NAMESPACE . '/v' . PINTEREST_FOR_WOOCOMMERCE_API_VERSION,
-				'optionsName'     => PINTEREST_FOR_WOOCOMMERCE_OPTION_NAME,
-				'error'           => isset( $_GET['error'] ) ? sanitize_text_field( wp_unslash( $_GET['error'] ) ) : '', // phpcs:ignore WordPress.Security.NonceVerification.Recommended --- not needed
-				'pinterestLinks'  => array(
-					'newAccount'       => 'https://business.pinterest.com/',
-					'claimWebsite'     => 'https://help.pinterest.com/en/business/article/claim-your-website',
-					'richPins'         => 'https://help.pinterest.com/en/business/article/rich-pins',
-					'enhancedMatch'    => 'https://help.pinterest.com/en/business/article/enhanced-match',
-					'createAdvertiser' => 'https://help.pinterest.com/en/business/article/create-an-advertiser-account',
-					'adGuidelines'     => 'https://policy.pinterest.com/en/advertising-guidelines',
-					'adDataTerms'      => 'https://policy.pinterest.com/en/ad-data-terms',
+				'createBusinessAccountUrl' => esc_url(
+					add_query_arg(
+						array(
+							'page' => 'wc-admin',
+							PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_go_to_create_account' => '1',
+							'view' => 'wizard',
+						),
+						admin_url( 'admin.php' )
+					)
+				),
+				'switchBusinessAccountUrl' => esc_url(
+					add_query_arg(
+						array(
+							'page' => 'wc-admin',
+							PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_go_to_switch_account' => '1',
+							'view' => 'wizard',
+						),
+						admin_url( 'admin.php' )
+					)
```

Comment:
For readability, these array values would be better as separate methods that return the appropriate value.